packages/sdk/src/authentication/bearerAuthProvider.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AxiosRequestConfig } from "axios";
+import { IAuthProvider } from "./iAuthProvider";
+
+/**
+ * Provider that handles Bearer Token authentication
+ *
+ * @beta
+ */
+export class BearerAuthProvider implements IAuthProvider {
+  private getToken: () => Promise<string>;
+
+  constructor(getToken: () => Promise<string>) {
```

Comment:
There could be some comments or code snippets to show how to leverage credentials provided by `TeamsFx` since we are accepting a `getToken` function here.

Reply:
Fixed.